src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -41,6 +42,72 @@ def persist_flow_run(self, run_info: FlowRunInfo):
         self.queue.put(run_info)
 
 
+class LineProcess:
+    def __init__(self, executor_creation_func):
+        self.process = None
+        self.input_queue = None
+        self.output_queue = None
+        self._executor_creation_func = executor_creation_func
+
+    def _new_process(self):
```

Comment:
We may use `start_new_process` for this, and do not return any values.
Then we use `is_ready()` to confirm whether it works.

Reply:
fixed